CHANGELOG.md

```diff
@@ -3,8 +3,63 @@ Change Log
 
 # Next version
 
-* Downloading or converting a SDL schema from introspection now includes scalar definitions. This is required for clients to get a [full view of the schema](https://github.com/graphql/graphql-wg/blob/main/rfcs/FullSchemas.md). 
-* The cache and auto persisted queries interceptors are now always added after all users interceptor. If you relied on some interceptors being called **after** `normalizedCache()` or `persistedQueries()`, you might have to update your code. One example is if you need to change cache flags based on the GraphQL response. In those cases, we recommend you use a custom `NetworkTransport` instead (See [this commit](https://github.com/apollographql/apollo-kotlin/pull/6455/commits/a53a44e5e506af7b0f6f495eed1a9d477e18bf73) for an example). 
+# Version 4.2.0
+
+_2025-04-28_
+
+## Generate custom scalars as inline classes with `@map` and `@mapTo` (#6404)
+
+It is now possible to configure the adapters to use with [scalars](https://www.apollographql.com/docs/kotlin/essentials/custom-scalars) by using the [`@map` directive](https://specs.apollo.dev/kotlin_labs/v0.5/#@map):
+
+```graphql
+extend schema @link(url: "https://specs.apollo.dev/kotlin_labs/v0.5/", import: ["@mapTo", "@map"])
+
+extend scalar Date @map(to: "kotlinx.datetime.Instant", with: "com.apollographql.adapters.InstantAdapter")
+
+## Scalar definitions in schemas downloaded from introspection (#6389)
+
+Downloading or converting an SDL schema from introspection now includes scalar definitions.
+This is required for clients to get a [full view of the schema](https://github.com/graphql/graphql-wg/blob/main/rfcs/FullSchemas.md).
+
+## Contributors 💜
+Many thanks to @bobbysothebys, @jvanderwee, @dhritzkiv, @lwasyl and @rohandhruva for all the contributions and help in this release 💜 
+
+## 👷‍♂️ All changes
+
+* [intellij-plugin] Fix MemoryCache package name (6383)
```

Comment:
I think PR numbers need a `#` in front of them to become clickable in the versions page

Reply:
Woops good catch 👍